docs/PORTING.md

```diff
@@ -117,12 +117,22 @@ int     flash_area_write(const struct flash_area *, uint32_t off,
 int     flash_area_erase(const struct flash_area *, uint32_t off, uint32_t len);
 /*< Returns this `flash_area`s alignment */
 uint8_t flash_area_align(const struct flash_area *);
-/*< Initializes an array of flash_area elements for the slot's sectors */
-int     flash_area_to_sectors(int idx, int *cnt, struct flash_area *ret);
-/*< Returns the `fa_id` for slot, where slot is 0 (primary) or 1 (secondary) */
-int     flash_area_id_from_image_slot(int slot);
-/*< Returns the slot, for the `fa_id` supplied */
-int     flash_area_id_to_image_slot(int area_id);
+/*< What is value is read from erased flash bytes. */
+uint8_t flash_area_erased_val(const struct flash_area *);
+/*< Reads len bytes from off, and checks if the read data is erased. */
```

Comment:
Sorry, have missed this one, but would be nice to also mention here that this function should return `1` for empty, `0` for not empty, and < 0 for errors.